types/index.d.ts

```diff
@@ -1,8 +1,22 @@
 /// <reference types="node" />
 
+/**
+ * Etablish a connection to a PostgreSQL server. 
```

Comment:
Wording change suggestion: `Etablish` -> `Establish`
I'm not native English speaker, but usually the word establish is used regarding database connections, also many dictionaries don't include the word etablish and suggest establish.

Reply:
I'm not a native English speaker either, proof! :laughing: